.bazelrc

```diff
@@ -10,6 +10,7 @@ build --compilation_mode=opt
 # Using C++ 17 on all platforms.
 build:linux --cxxopt="-std=c++17"
 build:macos --cxxopt="-std=c++17"
+build:macos --copt="-Wno-error=implicit-function-declaration"
```

Comment:
Could we do something like 
```
build --per_file_copt="external/.*@-Wno-unused-result"
build:linux --per_file_copt="-\\.(asm|S)$,external/com_github_grpc_grpc/.*@-DGPR_MANYLINUX1"
```

for only external dependencies? We could probably do that for the vendored thirdparty libs as well?